ember-app/app/components/hb-milestone-card.js

```diff
@@ -1,7 +1,43 @@
 import HbCard from "../components/hb-task-card";
+import Ember from "ember";
+
+var { get, set } = Ember;
+
+var takeWhile = function(list, callback, context) {
```

Comment:
Ah, I thought we were using lodash on the web. I know if I've seen it in an `npm install`, but I guess it's only a dependency of `ember-cli` stuff.